src/ng/browser.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
-/* global stripHash: true */
+/* global getHash: true, stripHash: false, trimEmptyHash: true */
```

Comment:
where is `getHash` used globally?

Reply:
It is "used" in tests, in a mock window. The feature that relies on it is not actually used atm, so there were no breakages.

See https://github.com/angular/angular.js/pull/16636/commits/34fe24f76e4b09c1a6574518eb6402a7a06c03ef (yes, there is a typo in that commit message :grin:).